src/skin/pixmapsource.cpp

```diff
@@ -48,6 +50,31 @@ void PixmapSource::setSVG(const QByteArray& content) {
     m_eType = SVG;
 }
 
+QPixmap PixmapSource::toPixmap(const QSize& size) {
```

Comment:
I first looked into using Paintable's draw methods inside WSpinny::paintEvent(), but yeah moving the toPixmap() function to Paintable and calling that from WSpinny::resizeEvent() seems like a better idea than the current state.